<launch>
    <node pkg="movie_publisher" name="camera" type="webcam_pub">
        <param name="camera_num" type="int" value="0" />
        <param name="camera_height" type="int" value="360" />
        <param name="camera_width" type="int" value="640" />
    </node>
    
    <node pkg="motpy_ros" name="face_tracking" type="face_tracking.py" output="screen">
        <param name="weights_url" value="https://github.com/opencv/opencv_3rdparty/raw/dnn_samples_face_detector_20170830/res10_300x300_ssd_iter_140000.caffemodel" />
        <param name="config_url" value="https://raw.githubusercontent.com/opencv/opencv/master/samples/dnn/face_detector/deploy.prototxt" />
        <param name="weights_path" value="/home/ubuntu/.cache/motpy_ros/face_tracking/opencv_face_detector.caffemodel"/>
        <param name="config_path"  value="/home/ubuntu/.cache/motpy_ros/face_tracking/deploy.prototxt"/>
        <param name="tracking_size/width" value="640" />
        <param name="tracking_size/height" value="480" />
    </node>
    <node pkg="motpy_ros" name="mosaic" type="mosaic_bbox.py"/>
 
</launch>